### DISCUSSION TITLE
Intro to PyTorch3D discussions

### DISCUSSION THREAD

**Discussion by nikhilaravi** [facebookresearch/pytorch3d#526] [General]
This forum is the best place for general "How do I do X?" type questions in PyTorch3D. Traditionally, we have handled such questions in Issues, but this forum will be more easily searchable by others with the same questions and will also help us to focus Issues on bug fixes and other urgent problems. 

We also welcome general feedback about PyTorch3D in this forum, or discussion of new PyTorch3D features that might help your work. 

Please obey the [Code of Conduct](https://github.com/facebookresearch/pytorch3d/blob/master/.github/CODE_OF_CONDUCT.md) and the GitHub Community Guidelines when discussing in this forum.

Thanks for your interest in PyTorch3D and for helping to make the codebase better!

**Comment by husnain-ali21** [comment]
Hello Is there a way to generate 3d mesh of an object from a single view image using just differential rendering?I know that we can generate 3d shapes given that we have multiple 2d views of the object but how would one do the same with just one view ?
